src/Mapper/Mapper.php

```diff
@@ -6,13 +6,13 @@
 
 interface Mapper
 {
-    public function canMap(object|string $objectOrClass, mixed $data): bool;
+    public function canMap(mixed $from, object|string $to): bool;
```

Comment:
I did a bunch of cleanup for mappers because it was a mess. Technically not related to this PR, but it needed to be done.